lib/registries/addon/drafts/draft/template.hbs

```diff
@@ -78,7 +78,10 @@
                         @navMode={{leftNav.leftGutterMode}}
                     />
                 </layout.leftNav>
-                <layout.main local-class='Main'>
+                <layout.main
+                    {{before-unload this.saveBeforeUnload}}
```

Comment:
This a somewhat weird case for using an element modifier because element modifiers expect to be passed an `element` on which event handlers are attached. In this case the element (whatever is yielded from `<layout.main>`) is not really needed, because we only need to attach a handler to the `beforeunload` event of the `window` element.

Reply:
Just a comment. I can't think of any better solutions.

Reply:
Yea, I had initially tried adding the event listener within a route lifecycle event, until I came across a post that suggested using a modifier. I'm hoping using the modifier makes it slightly easier to reuse, but we shall see...